### PR TITLE
Fix vectorpipeline which did not respect gm.linewidth

### PR DESCRIPTION
**Pull request by scottwittenburg**
Should hopefully address #381, but let's not close it until we get confirmation.

### CODE REVIEW

**Comment by scottwittenburg** [comment]
fyi @aashish24

**Comment by danlipsa** [comment]
@scottwittenburg Do you want to try my new packages and use vtkpython instead of python to this if this fixes mac py3?

1. use -c danlipsa before -c conda-forge

2. only for mac, py3, rename python to python_static and vtkpython to python.

**Comment by scottwittenburg** [comment]
Sure I can give this a try, thanks @danlipsa.

**Comment by scottwittenburg** [comment]
@danlipsa I pushed a second commit to try out what you suggested, can you take a look at it and see if that's what you were intending?

**Comment by scottwittenburg** [comment]
Ah shoot, I did the python swap for all cases, not just py3.  Let me try again.

**Comment by doutriaux1** [comment]
@scottwittenburg doing similar thing in my vcs_template branch...
We'll see who gets it right first ;)

**Comment by scottwittenburg** [comment]
I don't know if it was just adding @danlipsa's channel or what, but now CircleCI is failing to set up the environment.  I'm seeing:

```
Executing transaction: ...working... done
Collecting package metadata: ...working... WARNING conda.core.index:push_record(193): Skipping cdat/label/nightly/linux-64::libcdms-3.0.2018.06.12.22.33.ga415a0b-0 due to InvalidSpec: >1.10.*
done
Solving environment: ...working... failed

InvalidVersionSpec: Invalid version '>1.10.*': invalid operator with '.*'

Exited with code 1
```

Do you know what that's about @doutriaux1?

**Comment by doutriaux1** [comment]
I see that too. I suspect it is new conda…

Trying to force a conda a bit "ypunger"… for now, will rebuild libcdms once @nadeau4 gets to it I guess…

C.
